tests/provisioning.log

```diff
@@ -0,0 +1,45 @@
+Provisioning device...
+	Erasing secure storage... done.
+	Registering drivers... done.
+	Initializing PSA Crypto... done.
+	Registering factory-created keys... done.
+	Generating keys within the secure element... done.
+Device provisioned
+
+---------------------------------------------------------------------
+
+Device public keys:
+	Key ID 16:
+-----BEGIN PUBLIC KEY-----
+[0-9a-fA-F+/=]+
+[0-9a-fA-F+/=]+
+-----END PUBLIC KEY-----
+
+	Key ID 18:
+-----BEGIN PUBLIC KEY-----
+[0-9a-fA-F+/=]+
+[0-9a-fA-F+/=]+
```

Comment:
Is there a way to say that there can be an arbitrary number of lines? The exact length of the public key depends on a random factor (it's in a representation that strips leading zeros from numbers). We may be lucky enough that in this particular case the variation is within a range where the number of lines doesn't change, but even so this is brittle.

Reply:
I'm not sure if we can specify an arbitrary number of lines. However, there's this from https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-host-tests:
```
htrun matches lines from the compare log (on left) one by one. It keeps on looking for a line until it matches. Once matched it moves on to match the next line.
```
So if we know what the minimum number of lines the key could be, setting the log to match that many should work in all cases if I'm understanding it right.

Reply:
Oh, good. So as long as the number of matched lines is at most the minimum length of the PEM, which looks plausible, the matching is robust.